src/commands/force/user/create.ts

```diff
@@ -36,6 +37,36 @@ interface FailureMsg {
   message: string;
 }
 
+const permsetsStringToArray = (fieldsPermsets: string | string[]): string[] => {
+  if (!fieldsPermsets) return [];
+  return isArray<string>(fieldsPermsets)
+    ? fieldsPermsets
+    : fieldsPermsets.split(',').map((item) => item.replace("'", '').trim());
+};
+
+const standardizePasswordToBoolean = (input: unknown): boolean => {
+  if (typeof input === 'undefined' || input === null) {
+    return false;
+  }
+  if (typeof input === 'boolean') {
+    return input;
+  }
+  if (typeof input === 'string') {
+    if (input.toLowerCase() === 'true') {
+      return true;
+    }
+    if (input.toLowerCase() === 'false' || input === '') {
+      return false;
+    }
+  }
+  if (typeof input === 'number') {
+    if (input === 1) {
```

Comment:
@mshanemc I think this should be `0`.

Reply:
The n umber check block can be changed to
```
    if (typeof input === 'number') {
        return !!input;
    }
```

Reply:
This comes from either a command line vararg (in which case it's a string) or a json config file (in which case it's whatever the value is).  I'm pretty sure the docs don't say that 0 or 1 are valid values so we should only accept a boolean true and a string value lowercased of "true" to be true.  The rest is false.

Reply:
> This comes from either a command line vararg (in which case it's a string) or a json config file (in which case it's whatever the value is). I'm pretty sure the docs don't say that 0 or 1 are valid values so we should only accept a boolean true and a string value lowercased of "true" to be true. The rest is false.

What docs @shetzel?